console/src/main/java/org/georchestra/console/ws/backoffice/platform/InfosController.java

```diff
@@ -56,24 +56,18 @@ public class InfosController {
     @Setter
     private boolean competenceAreaEnabled;
 
-    @Value("${useLegacyHeader:false}")
-    private boolean useLegacyHeader;
-
-    @Value("${headerUrl:/header/}")
-    private String headerUrl;
-
-    @Value("${headerHeight:90}")
+    @Value("${headerHeight:80}")
     private String headerHeight;
 
     @Value("${headerScript:https://cdn.jsdelivr.net/gh/georchestra/header@dist/header.js}")
     private String headerScript;
 
-    @Value("${logoUrl:https://www.georchestra.org/public/georchestra-logo.svg}")
-    private String logoUrl;
-
     @Value("${georchestraStylesheet:}")
     private String georchestraStylesheet;
 
+    @Value("${headerConfigFile:}")
```

Comment:
shouldnt it have a valid default value ? or the default comes from the values hardcoded in the header source ?